components/brave_ads/browser/notification_helper.cc

```diff
@@ -38,7 +38,7 @@ NotificationHelper* NotificationHelper::GetInstance() {
   return GetInstanceImpl();
 }
 
-#if !defined(OS_MACOSX) && !defined(OS_WIN) && !defined(OS_LINUX) && !defined(OS_ANDROID)  // NOLINT
+#if !defined(OS_APPLE) && !defined(OS_WIN) && !defined(OS_LINUX) && !defined(OS_ANDROID)  // NOLINT
```

Comment:
Should be `OS_MAC`

Reply:
Addressed by 0230f8e014a698a5cf5abd780734aa5bae433b3f